pandemania/src/api/FetchAllCountries.js

```diff
@@ -0,0 +1,18 @@
+const apiUrl = 'https://api.covid19api.com/countries';
+
+const allCountries = async () => {
+	try {
+		const response = await fetch(apiUrl);
```

Comment:
I'd rather use .then() nstead of using await. That way you can be more confident the response is valid

Reply:
Done 👍